### PR TITLE
Sample imu bridge

### PR DESCRIPTION
**Pull request by mike-matera**
Here's some starter code with all the bells and whistles.

### CODE REVIEW

File: src/seahawk_rov/seahawk_rov/control_bridge.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        """Initialize this node"""
+        super().__init__('control_bridge')
+
+        # Publishers and pre-defined messages 
+
+        self.imu0_msg = Imu() 
+        self.imu1_msg = Imu() 
+        self.temp_msg = Temperature()
+        self.press_msg = FluidPressure()
+        self.humid_msg = RelativeHumidity()
```
Comment (cTurtle98) [inline src/seahawk_rov/seahawk_rov/control_bridge.py:29]: there is only 1 imu but there are two temp/pressure/humidity sensors

**Review by cTurtle98 — APPROVED** [review]
some good refrence information

File: src/seahawk_rov/seahawk_rov/control_bridge.py (hunk 1 above)
Comment (mike-matera) [reply src/seahawk_rov/seahawk_rov/control_bridge.py:29]: Doh! I misremembered that.

**Comment by cTurtle98** [comment]
I'll use this for reference for the ros parts for i2c_proxy and then i'll delete it.

I dont plan to actually use this node as code running on the robot

**Pull request merged**